unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/Channels/ChatChannelsControllerMock.cs

```diff
@@ -629,5 +635,69 @@ private void UpdateNearbyUnseenMessages(ChatMessage message)
                 controller.UpdateTotalUnseenMessages(JsonUtility.ToJson(totalUnseenMessagesPayload));
             }
         }
+
+        public void GetChannelInfo(string[] channelIds) =>
+            GetFakeChannelInfo(channelIds).Forget();
+
+        private async UniTask GetFakeChannelInfo(string[] channelIds)
+        {
+            await UniTask.Delay(Random.Range(40, 1000));
+
+            var msg = new ChannelInfoPayload
+            {
+                joined = true,
+                channelId = channelIds[0],
```

Comment:
what is the point of receiving an array of channel ids but only updating one?

Reply:
It was a mistake implementing the mocked flow. Fixed!

https://github.com/decentraland/unity-renderer/blob/e7e34e96114f9bd29f00ee7a356d4cb2e289d93d/unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/Channels/ChatChannelsControllerMock.cs#L184-L201